.github/workflows/ci.yml

```diff
@@ -17,21 +17,21 @@ jobs:
           node-version-file: .nvmrc
 
       # See https://github.com/actions/setup-node/issues/488
-      - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
-
-      - name: Cache yarn dependencies
-        uses: actions/cache@v3
-        id: yarn-cache
-        with:
-          path: ${{steps.yarn-cache-dir-path.outputs.dir }}
-          key: yarn-cache-${{ hashFiles('**/yarn.lock', '.yarnrc.yml') }}
-          restore-keys: |
-            yarn-cache-
+      # - name: Get yarn cache directory path
```

Comment:
Is disabling this necessary for this PR?

Reply:
In the course of re-triggering workflows on this PR a bunch, I was noticing the steps after the install step would sometimes repeatedly fail if the install step wasn't run. I'll try reverting this and see if it plays nicely again